flask/packages.py

```diff
@@ -114,3 +114,32 @@ def upload():
 
         return jsonify({"message": "Package Uploaded Successfully.", "code": 200})
 
+@app.route("/packages/<username>/<namespace_name>/<package_name>/delete", methods=["GET", "POST"])
+def delete_package(username, namespace_name, package_name):
+    uuid = request.cookies.get("uuid")
+    if not uuid:
+        return render_template("login.html")
```

Comment:
@arteevraina We would have to add a json response of unauthorized 403, as we have shifted to react/next.js frontend and for CLI application.